MonstieWash/Assets/Scripts/MonsterAi/MonsterController.cs

```diff
@@ -93,13 +93,35 @@ private void Attack()
         var attack = attackList[chosenAttack];
 
         // Get and save the animation that's being interrupted
-        var animatorInfo = this.m_myAnimator.GetCurrentAnimatorClipInfo(0);
+        var animatorInfo = m_myAnimator.GetCurrentAnimatorClipInfo(0);
         m_interruptedAnimation = animatorInfo[0].clip;
 
         // Play the attack animation
         m_myAnimator.Play(attack.name);
     }
 
+    private void Flinch()
+    {
+        if (m_interruptedAnimation != null)        // Another attack is already in progress
```

Comment:
Just one thing I noticed about the logic here, is that if the player is constantly causing the monstie to flinch, it will likely never attack the player since its animations will always be interrupted. We just invented stunlocking! Have a chat with Evita and ask what she thinks, might even be what the designers want, but just something to keep in mind

Reply:
Oh shoot true I didn't even think about that - I didn't want it to interrupt attacks while they were playing but I suppose this does also prevent attacks while this is playing. I'll try make some kinda priority thingo to rank them if the designers don't want stunlocking.